tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
The previous test was pulling values from a list. This should be faster than it was, and we only have to look at a single benchmark entry instead of 10.

These are all about relativity anyways. They only make sense when compared to prior runs or implementation differences.